src/components/field/field.scss

```diff
@@ -1,3 +1,5 @@
+@import '../encoding-pane/filter-shelf.scss';
```

Comment:
Why do we need to do an import here?  Can't we make the relevant component self-contained?